src/dataExplorer/shared/utils.test.ts

```diff
@@ -0,0 +1,18 @@
+import {sanitizeSQLSearchTerm} from 'src/dataExplorer/shared/utils'
+
+describe('test function - sanitizeSQLSearchTerm', () => {
+  it('should sanitize to valid search terms', () => {
+    const sanitizeMaps = {
+      '': '',
+      abcd: 'abcd',
+      "ab'cd": "ab''cd",
+      'ab%cd': 'ab\\%cd',
```

Comment:
why do you escape `%`?

And I think you need to escape `\` as well.

Reply:
Chatted offline with @crepererum and we found that `\` is not supported in sqlparser-rs, which is used in DataFusion and in IOx. Tracking issue https://github.com/sqlparser-rs/sqlparser-rs/issues/800

Because of this bug, users cannot search for a single `\`; however, if they search `\` with any character after it, e.g. `\<any_character>`, they will get what they want.

For now, I just leave the frontend code not escaping `\`. This is still safe and the SQL injection issue is already addressed, so not a big problem here